roles/linux_ansible_init/files/ansible-init.py

```diff
@@ -103,3 +110,13 @@ def ansible_exec(cmd, *args, **kwargs):
             "127.0.0.1,",
             playbook["name"]
         )
+
+
+logger.info("writing sentinel file /var/lib/ansible-init.done")
+SENTINEL = pathlib.Path("/var/lib/ansible-init.done")
+SENTINEL.parent.mkdir(mode = o755, parents = True, exist_ok = True)
```

Comment:
This needs to be `0o755`. Apologies - I think that was a typo in my original snippet.